docs/source/create/avatars/package-avatar.rst

```diff
@@ -404,16 +404,16 @@ The Avatar Packager will notify you of any errors or warnings that may affect th
             <tr>
                 <td>
                     <p id="unsupported-format" style="color: orange;"><strong>Unsupported avatar model format</strong></p>
-                    <p>This warning occurs when your avatar model is not a supported format. High Fidelity only supports FBX models for avatars.
+                    <p>This warning occurs when your avatar model is not a supported format. Vircadia only supports FBX models for avatars.
```

Comment:
```suggestion
                    <p>This warning occurs when your avatar model is not a supported format. Vircadia only supports FBX, glTF, and GLB models for avatars.
```
Pretty sure that the avatar packager has not been updated for this, but Vircadia supports glTF and GLB for avatars as well as FBX nowadays.